test/KHI_growthRate/include/picongpu/param/density.param

```diff
@@ -0,0 +1,46 @@
+/* Copyright 2013-2021 Axel Huebl, Heiko Burau, Rene Widera, Felix Schmitt,
```

Comment:
This is an old license header (year 2021). You can also add your name, as you prepared the setup.